server-phoenix/test/helios_web/controllers/web_hooks/github_controller_test.exs

```diff
@@ -0,0 +1,151 @@
+defmodule HeliosWeb.WebHooks.GithubControllerTest do
+  use HeliosWeb.ConnCase, async: true
+  use ExUnit.Case
+  import Ecto.Query
```

Comment:
This import isn't used anymore

Reply:
Leo is working through the warnings if you want to put this off. I'm fine with dealing with it here as well.